unittest/libmariadb/charset.c

```diff
@@ -645,7 +645,7 @@ static int test_bug_54100(MYSQL *mysql)
   {
     /* ignore ucs2 */
     if (strcmp(row[0], "ucs2") && strcmp(row[0], "utf16le") && strcmp(row[0], "utf8mb4") && 
-        strcmp(row[0], "utf16") && strcmp(row[0], "utf32")) {
+        strcmp(row[0], "utf16") && strcmp(row[0], "utf32") && strcmp(row[0], "gb18030")) {
```

Comment:
This test was failing for `gb18030` encoding and I saw that we ignore few more of the existing encodings too.

All these encodings are 4 byte encodings.

<img width="627" alt="Screen Shot 2020-04-06 at 2 15 48 PM" src="https://user-images.githubusercontent.com/51671/78591235-22180c80-7811-11ea-84e0-4e81acab4a09.png">